packages/trader/src/App/Containers/SettingsModal/settings-chart.jsx

```diff
@@ -50,17 +50,17 @@ const ChartSettings = ({
                         items={[
                             {
                                 label: <Localize i18n_default_text='Bottom' />,
-                                value: true,
+                                value: 'true',
```

Comment:
'true' doesn't feel right, why are we passing the string here?

Reply:
It does not work with boolean:
https://github.com/redux-form/redux-form/issues/3141

We have to either normalize or right strings:
https://github.com/redux-form/redux-form/issues/1243

Pkease let me know if there is another solution, I couldn't find any.

Reply:
@sara-fs you're right, @reza-fs this is much like creating radio buttons in pure HTML, your `value` attr. just takes in a string.